firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -155,10 +155,8 @@
 		1D3822E92BAB99250046BC5E /* UIView+ThemeUUIDIdentifiable.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1D3822E82BAB99250046BC5E /* UIView+ThemeUUIDIdentifiable.swift */; };
 		1D3C90882ACE1AF400304C87 /* RemoteTabPanelTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1D3C90872ACE1AF400304C87 /* RemoteTabPanelTests.swift */; };
 		1D4D79462BF2F4E7007C6796 /* SimpleTab.swift in Sources */ = {isa = PBXBuildFile; fileRef = 43E69EAF254D064E00B591C2 /* SimpleTab.swift */; };
-		1D4D79472BF2F4FD007C6796 /* Throttler.swift in Sources */ = {isa = PBXBuildFile; fileRef = 96D95015270238500079D39D /* Throttler.swift */; };
```

Comment:
@mattreaganmozilla `Throttler` was added to the `ClientTests` target membership and so `WindowSimpleTabsCoordinator` was as well. I didn't see why they should be added so I removed it. Let me know if any issues! 

<img width="263" alt="image" src="https://github.com/user-attachments/assets/f4b3a977-c239-4f60-94db-0c8c5a0f1a33" />